pom.xml

```diff
@@ -137,6 +137,7 @@
         <netty.version>4.1.52.Final</netty.version>
         <netty-tcnative-boringssl-static.version>2.0.34.Final</netty-tcnative-boringssl-static.version>
         <okhttp.version>3.14.6</okhttp.version>
+        <okta-sdk.version>3.0.1</okta-sdk.version>
```

Comment:
while this can't create any problems, I was wondering if there isn't a better place for this?

Reply:
@mpfz0r what do you have in mind as a better place? If I'm correct, the goal here is to resolve the dependency of two different plugins to the same version. As the Okta SDK is not used in Graylog core but only in plugins, it might make indeed sense to move it somewhere else. I see two options:
1. Add the version property definition to the plugin parent pom [here](https://github.com/Graylog2/graylog2-server/blob/092d6f7e53963b26ccfad43ad60b3afc3fdbfc6e/graylog-plugin-parent/pom.xml#L38) instead
2. Or manage the dependencies explicitly in the plugin parent pom [here](https://github.com/Graylog2/graylog2-server/blob/092d6f7e53963b26ccfad43ad60b3afc3fdbfc6e/graylog-plugin-parent/pom.xml#L50) and remove the version specifiers from dependencies in the plugin poms.

What do you think?

Reply:
Thanks for the feedback @mpfz0r. I am also curious if you have in mind any other particular place that might work better.

As far as I know, we currently only define common dependency versions in [graylog2-server/pom.xml](https://github.com/Graylog2/graylog2-server/blob/maser/pom.xml). I agree that, since this dependency is just used in plugins it perhaps _could_ be moved to the plugin-parent. But, we've not done this before with other plugins, so perhaps we need some feedback/agreement from others (eg. Bernd) if we really want start doing this.

Reply:
> @mpfz0r what do you have in mind as a better place?

None ;-) I'm always a bit lost when it comes to maven and our plethora of pom files.
But it kind of felt wrong putting this into the server...
IMHO the plugin parent pom sounds better. But I'll leave that up to you or @bernd

Reply:
@bernd Do you have an opinion on this?

Reply:
I will likely revert this, since we now cannot use the Okta Java SDK with the new Owner Resource Password authentication flow that we are using now to achieve the isolation between Cloud Okta users.

Reply:
The Okta User Management feature no longer uses the Okta SDK directly. However, the Enterprise Integrations Plugin uses it. I don't see an issue with leaving this here. This might be helpful if the Okta SDK is needed in another plugin in the future (since version conflicts are known to occur with it).

Reply:
Update: The `okta-sdk.version` has been removed, since we don't need it in this PR: https://github.com/Graylog2/graylog2-server/pull/9730/commits/88ab01b8cf85c43ae799e735a92748486521a176